README.md

```diff
@@ -1,7 +1,28 @@
-Dark-Dungeon
-============
+# Dark Dungeon
 
-To play Dark Dungeon, please download **DarkDungeonInstaller.zip**,
-extract its contents, then run the **setup.exe** file. This will
-install everything needed to play Dark Dungeon on your
-computer.
+## About the game
+Dark Dungeon is a 2D stealth puzzle game written in C# using Microsoft XNA. It was developed in 2014 for the University of Portland Game Design and Theory course.
+
+![title screen](https://lilah.io/img/Screenshots/Dark%20Dungeon/Dark%20Dungeon%20Title.png)
+
+![lights on](https://lilah.io/img/Screenshots/Dark%20Dungeon/Dark%20Dungeon%203.png)
+_Find your way to the treasure without being seen by the guards_
+
+![dark mode](https://lilah.io/img/Screenshots/Dark%20Dungeon/Dark%20Dungeon%202.png)
+_It's easier to sneak with the lights off but don't get caught by the ghosts!_
+
+## How to play
+Right now, Dark Dungon can only be played on Windows. To play it, please download `DarkDungeonInstaller.zip`, extract its contents, then run the `setup.exe` file. This will install everything needed to play Dark Dungeon on your computer.
```

Comment:
```suggestion
Currently, Dark Dungeon can only be played on Windows. To play it, please download `DarkDungeonInstaller.zip`, extract its contents, then run the `setup.exe` file. This will install everything needed to play Dark Dungeon on your computer.
```